### PR TITLE
[new] Add chartsly extjs library

### PR DESCRIPTION
**Pull request by ranjit-battewad**
Hi,

Please add new Stock Charts package based on Sencha Ext JS.

I am colleague of author, Ajit Kumar ( https://github.com/ajit-kumar-azad )

Repo : https://github.com/walkingtree/chartsly

Thank you.

### CODE REVIEW

File: ajax/libs/chartsly-extjs/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "bugs": {
+    "url": "https://github.com/walkingtree/chartsly/issues"
+  },
+  "author": {
+    "name": "Ajit Kumar",
+    "email": "ajit.kumar@walkingtree.in",
+    "url": "https://github.com/ajit-kumar-azad"
+  },
+  "homepage": "http://walkingtree.github.io/chartsly",
+  "license": "GNU General Public License version 3"
```
Comment (PeterDaveHello) [inline ajax/libs/chartsly-extjs/package.json]: Please use SPDX identifier(#5500)

File: ajax/libs/chartsly-extjs/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ 
+}
```
Comment (PeterDaveHello) [inline ajax/libs/chartsly-extjs/package.json]: Need a newline char here

File: ajax/libs/chartsly-extjs/package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "chartsly",
+    "senchacharts",
+    "extjs",
+    "stockcharts"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/walkingtree/chartsly.git"
+  },
+  "bugs": {
```
Comment (PeterDaveHello) [inline ajax/libs/chartsly-extjs/package.json]: Please remove bugs scope.

**Comment by PeterDaveHello** [comment]
I think this lib need much higher notable popularity.

**Comment by ranjit-battewad** [comment]
Updated package.json with review comments, please let me know if any more issues.

Thanks

**Comment by ranjit-battewad** [comment]
May I know what's the issue with my package.json file ?

**Comment by ranjit-battewad** [comment]
Hello @PeterDaveHello / @Piicksarn

Just wanted to know how much time it takes to publish library.


Thanks,

**Comment by PeterDaveHello** [comment]
Once it passed the review @ranjit-battewad.
Ping @Piicksarn for review ...

**Comment by ranjit-battewad** [comment]
Hi @Piicksarn can you please review this request and do needful, I am waiting for this to publish.

Thanks,

**Comment by Piicksarn** [comment]
@ranjit-battewad ,

There are some modification needed to the pr!
 -  For commits : 
    * Should pass the online test fist!
    * Should be rebased.
 - The license field in package.json should follow the link below for the  license name.

      https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json\

-  The needless field in package.json is shown below

     https://github.com/cdnjs/cdnjs/blob/master/tools/fixFormat.js

- The files structure need to follow the same structure with origin repo.

**There is a check list for adding a library**
```
  - Not found on cdnjs repo
  - No already exist issue and PR
  - Notable popularity : **Watch `xxx`, Star `xxx`, Fork `xxx`**
  - Project has public repository on famous online hosting platform
    (or been hosted on npm) 
  - Has valid tags for each versions (for git auto-update)
  - For auto-update : 
     * Auto-update setup
     * Auto-update target/source is valid.
     * Auto-update filemap is correct.
```

Thank you!
Piicksarn.

**Comment by ranjit-battewad** [comment]
Thanks @Piicksarn.

Please review my pull request and let me know what all changes I need to make  in order to get this published ASAP.

**Comment by Piicksarn** [comment]
@ranjit-battewad, any update?

**Comment by drewfreyling** [comment]
@ranjit-battewad There are still issues already listed above for you to fix.

**Comment by PeterDaveHello** [comment]
The most important issue is its popularity.

**Comment by PeterDaveHello** [comment]
Close due to no response.

**Pull request closed without merging**